test/index.test.js

```diff
@@ -67,44 +67,48 @@ describe('SatisMeter', function() {
       it('should send apiKey and user id', function() {
         analytics.identify('id');
         analytics.called(window.satismeter, {
-          token: options.apiKey,
-          user: {
-            id: 'id'
-          }
+          writeKey: options.apiKey,
+          userId: 'id',
+          traits: {
+          },
+          type: 'identify'
         });
       });
 
       it('should send email', function() {
         analytics.identify('id', { email: 'email@example.com' });
         analytics.called(window.satismeter, {
-          token: options.apiKey,
-          user: {
-            id: 'id',
+          writeKey: options.apiKey,
+          userId: 'id',
+          traits: {
             email: 'email@example.com'
-          }
+          },
+          type: 'identify'
         });
       });
 
       it('should send user name', function() {
         analytics.identify('id', { name: 'john doe' });
         analytics.called(window.satismeter, {
-          token: options.apiKey,
-          user: {
-            id: 'id',
+          writeKey: options.apiKey,
+          userId: 'id',
+          traits: {
             name: 'john doe'
-          }
+          },
+          type: 'identify'
         });
       });
 
       it('should send signUpDate', function() {
         var now = new Date();
-        analytics.identify('id', { created: now });
+        analytics.identify('id', { createdAt: now });
```

Comment:
createdAt is standard trait, as specified by your spec:
https://segment.com/docs/spec/identify/#traits